data/json/mutations.json

```diff
@@ -810,11 +810,20 @@
         "type" : "mutation",
         "id" : "SCHIZOPHRENIC",
         "name" : "Schizophrenic",
-        "points" : -5,
+        "points" : -3,
         "description" : "You will periodically suffer from delusions, ranging from minor effects to full visual hallucinations.  Some of these effects may be controlled through the use of Thorazine.",
         "starting_trait" : true,
         "valid" : false,
         "category" : ["MEDICAL"]
+    },{
+        "type" : "mutation",
+        "id" : "NARCOLEPTIC",
+        "name" : "Narcoleptic",
+        "points" : -2,
```

Comment:
An additional point may be warranted (to -3) as the risk of the falling asleep while driving can be game-ending.